### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by DavyKn**
- [ ] Project made using HTML and JS only
- [ ] objects can be added and deleted using the ADD and REMOVE button.
- [ ] The objects can be stored in the LOCAL STORAGE.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  bookDiv.append(bookTitle, bookAuthor, AremoveBtn, Ahr);
+  bookContainer.appendChild(bookDiv);
+
+  AremoveBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const remaining = books.filter((book, bookIndex) => bookIndex !== index);
+    localStorage.setItem('books', JSON.stringify(remaining));
+    bookDiv.remove();
+  });
```
Comment (medaminedev66) [inline script.js]: - [ ]  When I click the remove button to remove a certain book, the book is not removed from the localStorage. Please make sure the local storage is updated after removing a specific book. You may use the dev tools to check if the right book is removed from the local storage.

In the screenshot below, the two books I deleted are still in the local storage

image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- There are no linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  bookContainer.appendChild(bookDiv);
+
+  AremoveBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    bookDiv.remove();
+  });
+};
+
```
Comment (sja-thedude) [inline script.js]: - [ ] As suggested previously when the remove button is clicked to remove a certain book, the book is not removed from the localStorage. Currently, one book is displayed in the local storage.
In the video below, one book is persistent in the local storage. Kindly fix it.

https://user-images.githubusercontent.com/70269507/176153902-319986d8-6e18-4575-b9ca-d091ff3686bb.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi Team, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**